src/Transports.AspNetCore/IUserContextBuilder.cs

```diff
@@ -1,16 +1,17 @@
 using Microsoft.AspNetCore.Http;
 
-namespace GraphQL.Server.Transports.AspNetCore;
-
-/// <summary>
-/// Interface which is responsible of building a UserContext for a GraphQL request
-/// </summary>
-public interface IUserContextBuilder
+namespace GraphQL.Server.Transports.AspNetCore
 {
     /// <summary>
-    /// Builds the UserContext using the specified <see cref="HttpContext"/>
+    /// Interface which is responsible of building a UserContext for a GraphQL request
     /// </summary>
-    /// <param name="httpContext">The <see cref="HttpContext"/> for the current request</param>
-    /// <returns>Returns the UserContext</returns>
-    Task<IDictionary<string, object>> BuildUserContext(HttpContext httpContext);
+    public interface IUserContextBuilder
+    {
+        /// <summary>
+        /// Builds the UserContext using the specified <see cref="HttpContext"/>
+        /// </summary>
+        /// <param name="httpContext">The <see cref="HttpContext"/> for the current request</param>
```

Comment:
```suggestion
        /// <param name="httpContext">The <see cref="HttpContext"/> for the current request.</param>
```

Reply:
This entire file had not been updated to my copy.  I have done so; see updated comments.